### ISSUE TITLE
Translate upload API error

### ISSUE THREAD

**Issue by NicolasMahe** [liteflow-labs/liteflow-js#51]
Original Trello card: https://trello.com/c/9GnFunrD/151-unlockable-content-translate-upload-api-error

Need to translate error returned by API

Currently only 2 errors can be easily translated as they are "key" based rather than sentence based.

Related to last step of https://trello.com/c/0YQMZLV4/125-onlyshoot-unlockable-content-ux-improvements-to-prevent-the-creation-of-nfts-that-have-already-being-minted-as-unlockable-conten

- Need to find a proper way to translate those error. If I take `file_is_already_public` and `file_is_already_private` errors, they are inside another error with a different message. Should we create a new error class that have the translation key explicitly defined?

- Note on errors `file_is_already_public` and `file_is_already_private`:
Need some discussion about how to do it properly. The best is for the error to be showed directly in the form in the right dropzone.

Once this issue is done, we should expand this system to all API error. Need to create a new card for this point

- [ ] `file_is_already_public`
- [ ] `file_is_already_private`
